parfive/conftest.py

```diff
@@ -5,6 +5,14 @@
 from parfive.tests.localserver import MultiPartTestServer, SimpleTestServer, error_on_nth_request
 
 
+@pytest.fixture
+def namedserver():
```

Comment:
why named?

Reply:
Because it serves files with the names it's given. It's a terrible name.